Cython/Compiler/ExprNodes.py

```diff
@@ -13446,6 +13471,15 @@ def calculate_result_code(self):
                 method,
                 operand2.result(),
                 operand1.result())
+        elif self.operator in ['in', 'not_in']:
+            if self.operator == 'not_in':
+                negation = "!"
+            else:
+                negation = ""
+            return "(%s__operator_in_containers_helpers__::is_in(%s,%s))" % (
+                negation,
+                operand1.result(),
+                operand2.result())
```

Comment:
Let's keep this simple.
```suggestion
            return "(%s __operator_in_containers_helpers__::is_in(%s,%s))" % (
                "!" if self.operator == 'not_in' else "",
                operand1.result(),
                operand2.result())
```